docs/content/tutorial/0-set-up-your-development-environment.mdx

```diff
@@ -210,6 +210,24 @@ You can download PostgreSQL on your macOS using [the installer on their website]
 </TabItem>
 </Tabs>
 
+#### Set up PostgreSQL with Docker
```

Comment:
can you instead add a docker tab to the list of tabs above? so it will be one of the tabs next to macOS, windows, etc...

Reply:
Done as you requested. Please tell me if it needs further fine-tuning.